lib/platform.c

```diff
@@ -542,21 +543,22 @@ int STDCALL sf_delete_directory_if_exists(const char * directoryName)
 
 void STDCALL sf_get_tmp_dir(char * tmpDir)
```

Comment:
Can you add comments to the function declaration informing user the minimum memory required for tmpDir?

Reply:
Left the original funtion as is and created a new function sf_get_uniq_tmp_dir which has comments.